src/routes/components/TextArea.tsx

```diff
@@ -0,0 +1,248 @@
+import {useState} from "react";
+import {ComponentBinding, Sandbox} from "@components/sandbox";
+import {
+  ComponentProperties,
+  ComponentProperty,
+} from "@components/component-properties/ComponentProperties.tsx";
+import {Category, ComponentHeader} from "@components/component-header/ComponentHeader.tsx";
+import {
+  GoABadge,
+  GoABlock,
+  GoAButton,
+  GoAButtonGroup,
+  GoAContainer,
+  GoADetails,
+  GoAFormItem,
+  GoATab,
+  GoATabs,
+  GoATextArea,
+} from "@abgov/react-components";
+import {CodeSnippet} from "@components/code-snippet/CodeSnippet.tsx";
+
+export default function TextAreaPage() {
+  const [textAreaProps, setTextAreaProps] = useState({
```

Comment:
Now that all the prop types are exported from the React lib we should try to use them. See the _template.tsx file for a better idea where they are used.